### ISSUE TITLE
Gallium Nine installed from Protontricks broken

### ISSUE THREAD

**Issue by Kiwii** [Tk-Glitch/PKGBUILDS#214]
Nine installed into a game's prefix through protontricks makes the game crash on startup when selecting Proton-tkg as Steam Play compatibility tool.

    002a:err:module:import_dll Library d3d9.dll (which is needed by L"Z:\\opt\\games\\steamapps\\common\\HatinTime\\Binaries\\Win64\\HatinTimeGame.exe") not found
    002a:err:module:LdrInitializeThunk Importing dlls for L"Z:\\opt\\games\\steamapps\\common\\HatinTime\\Binaries\\Win64\\HatinTimeGame.exe" failed, status c0000135

Upon looking into it, it seems .../pfx/drive_c/windows/system32/d3d9.dll gets replaced with a symlink to .local/share/Steam/compatibilitytools.d/proton_tkg_4.9.r5.g8725e273/dist/lib64/wine/fakedlls//d3d9.dll upon every start of the game.

I tried setting PROTON_USE_WINED3D9 in user_settings.py to both 1 and 0 but it didn't seem to help.

I also tried setting _use_pba="false" and it made no difference.

I believe this is different from the global Wine regression mentioned in #189, as 

    protontricks -c "wine ninewinecfg" 253230

 runs fine and creates the correct symlinks and it's only when starting the game that the links are being broken (even when I replace them with a copy of the actualy d3d9-nine.dll). 

I assume I'm doing something wrong with the build config, so here's my [last_build_config.log](https://github.com/Tk-Glitch/PKGBUILDS/files/3235133/last_build_config.log)

**Comment by Tk-Glitch** [comment]
D9VK support interferes with G9.
`PROTON_USE_WINED3D9=0` will use d3d9.dll from `proton_tkg_4.9.r5.g8725e273/dist/lib*/wine/d9vk/`
`PROTON_USE_WINED3D9=1` will use d3d9.dll from `proton_tkg_4.9.r5.g8725e273/dist/lib*/wine/fakedlls/`
Putting G9 d3d9.dll in your game directory(if the game works with that) or replacing D9VK ones in `proton_tkg_4.9.r5.g8725e273/dist/lib*/wine/d9vk/` would work.

**Comment by Kiwii** [comment]
Thanks for the info. I symlinked d3d9-nine.dll into the gamefolder (as d3d9.dll) and that does indeed work.

Yes, I did build it with _use_d9vk="false", which is why I didn't expect this behaviour. Looking at proton_template/conf/proton, I see now where it's coming from.

Imho it would be cool if this would be disabled when built without d9vk but I understand it would make everything a little more complicated for pretty little gain.
